### PR TITLE
Disable experimental components by default

### PR DESCRIPTION
**Pull request by flands**
- Disable metric collection
- Add experimental banner to relevant pages

Also alphabetize table env vars

### CODE REVIEW

**Comment by jkwatson** [comment]
I'm curious why we would want to disable the server-timing header. Is it harmful to have it enabled by default? I would think it has near zero impact, in general.

**Comment by iNikem** [comment]
+1 to John's question. This was part of epic's acceptance criteria to have it enabled by default, if I am not mistaken

File: docs/advanced-config.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ Below you will find all the configuration options supported by this distribution
...
+| System property                        | Environment variable                   | Default value           | Support | Purpose                                                                                                                                                                                                                                                          |
+| -------------------------------------- | -------------------------------------- | ----------------------- | ------- | -------                                                                                                                                                                                                                                                          |
+| `splunk.access.token`                  | `SPLUNK_ACCESS_TOKEN`                  | unset                   | Beta  | (Optional) Auth token allowing exporters to communicate directly with the Splunk cloud, passed as `X-SF-TOKEN` header. Currently the [both Jaeger and OTLP trace exporters](#trace-exporters) and [SignalFx metrics exporter](metrics.md) support this property.
```
Comment (mateuszrzeszutek) [inline docs/advanced-config.md:16]: Is `splunk.access.token` beta too? I thought that it was a stable feature.

Also, should we use word "beta" or "experimental"? I think we use "experimental" for all other features, the whole project is in "beta".

**Comment by mateuszrzeszutek** [comment]
> I'm curious why we would want to disable the server-timing header. Is it harmful to have it enabled by default?

It's also required to be true by the GDI spec. Should we change the spec first?

File: docs/advanced-config.md (hunk 1 above)
Comment (flands) [reply docs/advanced-config.md:16]: Idea is to mark stable with 1.0 release. Experimental is likely a better term -- will update.

**Comment by flands** [comment]
> I'm curious why we would want to disable the server-timing header. Is it harmful to have it enabled by default? I would think it has near zero impact, in general.

With goal to move this to OTel, thought was to not mark stable so easy to remove. Not marking stable means off by default. Sounds like this may take longer than expected so will opt to mark stable/on and we can deprecate when OTel is ready (per GDI spec)

**Review by mateuszrzeszutek — APPROVED** [review]

**Pull request merged**